src/pages/Scan.vue

```diff
@@ -144,6 +152,15 @@ export default {
             }, 2000);
         },
 
+        showQueued: function() {
+            this.spinner = false;
+            this.queued = true;
+            var self = this;
+            setTimeout(function(){
```

Comment:
Don't like arbitrary timeouts. Could we add a comment in here to explain why we need it?